### PR TITLE
Evaluate method for additional unary operators

### PR DESCRIPTION
**Pull request by Priya2698**
Evaluate method for `UnaryOp::Tanh`, `UnaryOp::Sigmoid`, `UnaryOp::Relu`, `UnaryOp::Gelu`, `UnaryOp::Exp`.

### CODE REVIEW

**Comment by Priya2698** [comment]
!build

File: test/test_gpu2.cpp [hunk 1]
```diff
@@ -8679,7 +8632,7 @@ TEST_F(NVFuserTest, FusionFloatPow_CUDA) {
       &fusion,
       outputs,
       aten_inputs,
-      {p4, p2, p2, p3, p3, t6},
+      // {p4, p2, p2, p3, p3, t6},
```
Comment (zasdfgbnm) [inline test/test_gpu2.cpp:8635]: Remove dead code

Comment (Priya2698) [reply test/test_gpu2.cpp]: Restored this, this should not be commented.

**Comment by Priya2698** [comment]
!build

**Review by zasdfgbnm — APPROVED** [review]

**Pull request merged**